rpgs/models.py

```diff
@@ -32,6 +32,9 @@ class Rpg(models.Model):
     member_only = models.BooleanField(default=False,
                                       help_text="Require users to be verified members to sign up")
     messaging_thread = models.ForeignKey(MessageThread, on_delete=models.SET_NULL, null=True, blank=True)
+    sent_notif = models.BooleanField('Should publishing this event notify everyone?', default=True,
+                                      help_text="It is recommended that this should be ticked, since notifying members will help you get players.\n" +
```

Comment:
I'm not sure \n will work due to html smooshing whitespace. `<br>` should work though.

Reply:
fixed via fa38d11